scripts/AbinsModules/LoadGAUSSIAN.py

```diff
@@ -177,3 +183,24 @@ def _read_atomic_disp_block(self, file_obj=None, disp=None):
             l = file_obj.readline().split()
             num_atom += 1
         self._num_read_freq += freq_per_line
+
+    def _read_masses_from_file(self, file_obj=None):
+        masses = []
+        pos = file_obj.tell()
+        self._parser.find_first(file_obj=file_obj, msg="Thermochemistry")
+
+        end_msg = "Molecular mass:"
+        key = "Atom"
+        if six.PY3:
```

Comment:
[This will break in python 4.](http://astrofrog.github.io/blog/2016/01/12/stop-writing-python-4-incompatible-code/)

Reply:
@quantumsteve  OK I will rearrange it to  check for python 2 and make python3 default.  I have it in few places I will also change it there.